src/core/obj.js

```diff
@@ -28,9 +28,10 @@ import { CipherTransformFactory } from './crypto';
 import { ColorSpace } from './colorspace';
 
 var Catalog = (function CatalogClosure() {
-  function Catalog(pdfManager, xref, pageFactory) {
+  function Catalog(pdfManager, xref, pageFactory, pdfFunctionFactory) {
     this.pdfManager = pdfManager;
     this.xref = xref;
+    this.pdfFunctionFactory = pdfFunctionFactory;
```

Comment:
I cannot spot why we need this property here.

Reply:
> I cannot spot why we need this property here.

We don't, that's just a case of a bit too much copy-pasting. Removed now, thanks!